src/docs/references/marketplace-assets/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Marketplace assets
+slug: marketplace-assets
+updated: 2022-05-09
+category: references
+ingress:
+  Reference documentation providing information on Marketplace assets.
+published: true
+---
+
+_Marketplace assets_ are a mechanism defining configuration data and
+content for a marketplace. They are most useful for storing client
+application configuration and content that is managed by a marketplace
+operator and changes relatively infrequently. Each _marketplace asset_
+is an object that defines a _path_ and _content_. The path gives the
+asset a name and allows organizing assets in a way similar to a file
+system, where assets can be organized in directory-like structure. The
+content of the asset is its data in one of the supported data types.
+
+At present, Flex supports only JSON data as assets. Support for images
+(JPEG, PNG, etc) is coming in the future.
+
+The marketplace assets are typically edited by a marketplace operator
+though Flex Console.
+
+**IMPORTANT** All marketplace assets are considered public and must not
+be used to store secret or otherwise sensitive information.
+
+> **Note:** Marketplace assets are in early development. At present,
+> Console allows only limited ability to manage the content of a single
+> marketplace asset - the JSON document describing the
+> [translation strings](/concepts/translations/) for a client
+> application. Stay tuned for more later on!
+
+For example, a marketplace may have assets with the following paths:
+
+```
+config/config.json
+config/translations.json
+```
+
+where `config/config.json` could contain configuration data (for
+instance, UI colors, marketplace pricing settings, etc) and
+`config/translations.json` could contain string translation data for use
+in a marketplace client application (such as one based on one of the FTW
+templates).
+
+## Asset versioning
+
+Collectively, the set of assets of a marketplace is called the
+_marketplace asset tree_. Each time the data of one or more assets is
+changed, a new asset is added, or an existing asset is deleted, a new
+version of the asset tree is created. Old asset tree versions are never
+updated. In other words, tree versions are immutable.
+
+Each tree version, therefore, represents the exact set of assets and
+their data at the time when the version was created.
+
+To facilitate access to the latest asset data, Flex maintains a built-in
+_alias_ called `latest` that always refers to the latest asset tree
+version.
+
+Old versions of an assets may be automatically deleted, but no sooner
```

Comment:
Wait, what does this mean?

Reply:
I wanted to have something that explains that we don't promise that we keep old versions indefinitely. I.e. we may delete old unaliased versions, etc. But we need to still promise they exist for at least a while, so 24 hours sounded like an ok level. What do you think, @rap1ds?